scripts/infra/cloud-instance.sh

```diff
@@ -64,7 +64,7 @@ ec2__launch() {
         --security-group-ids "$EC2_SECURITY_GROUP_ID" \
         --subnet-id "$EC2_SUBNET_ID" \
         --key-name "$EC2_KEY_NAME" \
-        --block-device-mappings '{"DeviceName": "/dev/sda1","Ebs": {"VolumeSize": 300}}' \
+        --block-device-mappings '{"DeviceName": "/dev/sda1","Ebs": {"VolumeSize": 600}}' \
```

Comment:
FYI this increase is because we bumped the ami awhile back to 600, so will fail otherwise